core/consensus/event.hpp

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright Soramitsu Co., Ltd. 2016 All Rights Reserved.
+http://soramitsu.co.jp
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#ifndef CORE_CONSENSUS_EVENT_HPP_
+#define CORE_CONSENSUS_EVENT_HPP_
+
+#include <string>
+
+#include "../service/json_parse.hpp"
+
+namespace event{
+
+class Event {
+public:
+  int order = 0;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The class 'ConsensusEvent' defines member variable with name 'order' also defined in its parent class 'Event'.](https://www.codacy.com/app/soramitsu/iroha/file/4478582988/issues/source?bid=3712491&fileBranchId=3923843#l28)